roles/docker/tasks/main.yml

```diff
@@ -75,9 +75,9 @@
     enabled: true
     daemon_reload: true
 
-- name: Wait for 60 seconds before commencing
+- name: Wait for 40 seconds before commencing
   ansible.builtin.wait_for:
-    timeout: 60
+    timeout: 40
```

Comment:
Argument for this particular value? Maybe it makes more sense to add a variable you can tweak using the inventory.

Reply:
https://github.com/saltyorg/Saltbox/commit/aaa8eba491f68e66ad31f9b62175f149e43f53b3 this should allow you to override the default delay